boards/arm/adi_eval_adin2111ebz/doc/index.rst

```diff
@@ -0,0 +1,177 @@
+| GPIO         | on-chip    | gpio                                |
++--------------+------------+-------------------------------------+
+| I2C          | on-chip    | i2c                                 |
++--------------+------------+-------------------------------------+
+| SPI          | on-chip    | spi                                 |
++--------------+------------+-------------------------------------+
+| PWM          | on-chip    | pwm                                 |
++--------------+------------+-------------------------------------+
+| WATCHDOG     | on-chip    | independent watchdog                |
++--------------+------------+-------------------------------------+
+| ADIN2111     | spi        | adin2111 10BASE-T1L mac/phy         |
++--------------+------------+-------------------------------------+
+| FT232        | uart       | usb-uart                            |
++--------------+------------+-------------------------------------+
+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/adi_eval_adin2111ebz/adi_eval_adin2111ebz_defconfig``
+
+
+Connections and IOs
+===================
+
+ADI ADIN2111EBZ evaluation board has 7 GPIO controllers (from A to G).
+These controllers are responsible for pin muxing, input/output, pull-up, etc.
+
+For mode details please refer to `EVAL-ADIN2111EBZ User Guide <https://wiki.analog.com/resources/eval/user-guides/eval-adin2111ebz-user-guide>`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_1 TX/RX : PA9/PA10 (UART to FT232, console)
+- SPI1 SCK/MISO/MOSI : PA5/PA6/PA7 (SPI to external nor flash IS25LP128)
+- SPI2 SCK/MISO/MOSI : PB13/PB14/PB15 (SPI to external ADIN2111)
+- LED1 : POWER (Green LED)
+- UC_LED1 : PB6 (Blue LED)
+- MOD LED1 : PE2 (SR LED)
+- MOD LED2 : PE6 (BG LED)
+- NET LED1 : PB10 (SR LED)
+- NET LED2 : PB11 (BG LED)
+
+
+System Clock
+------------
+
+EVAL-ADIN2111EBZ System Clock could be driven by an internal or external oscillator, as well as the
+main PLL clock. By default the System clock is driven by the PLL clock at 80MHz, driven by the
+16MHz high speed internal oscillator.
+
+Serial Port
+-----------
+
+EVAL-ADIN2111EBZ has 1 U(S)ART. The Zephyr console output is assigned to UART1 that is connected
+to a FT232, avialable through Micro USB connector. Default settings are 115200 8N1.
+Same UART1 TX and RX cmos signals are available before the FT232, at P9 connector.
+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+EVAL-ADIN2111EBZ includes an ST-LINK/V2-1 JTAG/SWD 10 or 20 pin connector. This interface is
+supported by the openocd version included in Zephyr SDK.
+
+Flashing an application to  Discovery kit
+-------------------------------------------------------
+
+Connect the EVAL-ADIN2111EBZ to your host computer using the USB port, then run a serial host
+program to connect with your ADI board. For example:
+
+.. code-block:: console
+
+   $ minicom -D /dev/ttyACM0
```

Comment:
```suggestion
   $ minicom -D /dev/serial/by-id/usb-usb-ADI_EVAL-ADIN1110EBZ_AVAS_XXXXXX-if00-port
```

Reply:
sorry, i cannot see that path, anyway, "minicom -D /dev/ttyACM0" is a standard command in all boards, generic since no one can know any indexing value.

Reply:
Ok, set the pat as /dev/serial